install_git-cinnabar.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+curl https://raw.githubusercontent.com/glandium/git-cinnabar/master/download.py -o download.py
+chmod u+x download.py
+./download.py --exact 0.7.0beta2
```

Comment:
There's a version 7.2 now.

Reply:
@shtrom I copied this from [the `git-hg-sync` repo](https://github.com/mozilla-conduit/git-hg-sync/blob/main/install_git-cinnabar.sh), we may want to update this there as well.

Reply:
I know you did (:

and, yes.